sniffer.py

```diff
@@ -150,7 +156,23 @@ def main():
                 pkt = result.value[2:2+length]
                 if options.ti_crc:
                     pkt = ti_crc(pkt)
-                pkt = pcap.encode_frame(pkt)
+
+                # metadata format:
+                # 0. RSSI(int8)
+                # 1. Noise Floor(int8)
+                # 2. Flags(uint16)
+                # 3. PHY-specific data struct contains:
```

Comment:
I put the timestamp in "[PHY-specific data](https://github.com/openthread/openthread/blob/master/doc/draft-rquattle-spinel-unified.txt#L1929)" instead of standard fields, since I think the timestamp is only used by sniffer (one use case of PROP_STREAM_RAW), but not required by the other properties like PROP_STREAM_NET. So we can limit this feature within OPENTHREAD_ENABLE_RAW_LINK_API only. What do you think?